html2asketch/group.d.ts

```diff
@@ -0,0 +1,12 @@
+import Base from './base';
+
+export interface GroupProps {
+  x: number;
+  y: number;
+  width: number;
+  height: number;
+}
+
+export default class Group extends Base {
+  constructor(props: GroupProps);
+}
```

Comment:
let's see if we can validate these types - see if they match the classes they are discribing